CHANGELOG.md

```diff
@@ -38,6 +38,11 @@ on timeout, try increasing the value, for example, twice. Also note that the
 execution of commands with the `--await` flag and without an explicitly
 specified time period is now limited to 1 minute. This value can be changed with
 `--timeout` flag.
+Histogram (not counter) RPC/engine operation handling time metrics were added. For
```

Comment:
That's `migration`, but something should also be written into the `added` section.